package.json

```diff
@@ -24,6 +24,8 @@
     "node": ">=8.0.0"
   },
   "dependencies": {
+    "@emotion/core": "^10.0.5",
+    "@emotion/styled": "^10.0.5",
     "babel-plugin-styled-components": "^1.7.1",
     "classnames": "^2.2.6",
     "emotion": "^9.2.9",
```

Comment:
You can remove the old base `emotion` package as well.

Reply:
Oh, and don't forget to upgrade `gatsby-plugin-emotion` to the latest version (`^4.0.1`) as well.